llama-rs/src/llama.rs

```diff
@@ -577,7 +611,12 @@ impl LlamaModel {
                 }
 
                 n_tensors += 1;
+                if n_tensors % 8 == 0 {
+                    print!(".");
+                    std::io::stdout().flush();
+                }
             }
+            println!(".");
```

Comment:
Brought this back temporarily because I was going crazy without it :smile:  I'll just end up rebasing on whatever we agree for as part of #10.